CI/travis.osx.after_success.sh

```diff
@@ -2,9 +2,8 @@
 
 set -e
 
-if [ "$TRAVIS_EVENT_TYPE" = "cron" ]; then
-  echo Job not executed under cron run
-  exit
+if [[ "${MUDLET_VERSION_BUILD}" == -public-test-build* ]]; then
```

Comment:
shouldn't both sides of the comparison be inside double quotes?

Reply:
Apparently no... internet says that'll make the right hand side be a literal match.

Reply:
Huh, well there are exceptions for everything. 9 times out of 10 if a comparison gives me crap in bash it's cuz I forgot the quotes, not left them on when I shouldn't have. Carry on then.